frontend/src/queries/nodes/DataTable/renderColumnMeta.tsx

```diff
@@ -46,7 +46,7 @@ export function renderColumnMeta(key: string, query: DataTableNode, context?: Qu
             />
         )
     } else if (key.startsWith('context.columns.')) {
-        const column = trimQuotes(key.substring(16))
+        const column = trimQuotes(key.substring(16)) // 16 = "context.columns.".length
```

Comment:
🤣 I read this as python's integer division and was thinking "wtf is happening here?!" 🤣